packages/react-instantsearch-hooks-dom/src/ui/__tests__/Breadcrumb.test.tsx

```diff
@@ -197,10 +197,10 @@ describe('Breadcrumb', () => {
       className: 'MyCustomBreadcrumb',
       classNames: {
         root: 'ROOT',
-        rootNoRefinement: 'ROOTNOREFINEMENT',
+        noRefinementRoot: 'ROOTNOREFINEMENT',
```

Comment:
Would it make sense to rename the default class to fit the new class name? Not required, but would be consistent.

Reply:
I did not want to add too much noise with the snapshots diff it will generate, but I can make the changes in a single commit so it can be viewed independently.